### DISCUSSION TITLE
Metric question - wafl_read_io_type

### DISCUSSION THREAD

**Discussion by JerryCurl** [NetApp/harvest#1384] [Q&A] [answered]
On the Node dashboard, under the backend drilldown, there is a graph that shows Reads From: 

There are two keys there that might be interesting for us - "cloud" and "cloud_s2c".  Anyone know what these are?  We just installed a StorageGrid and have FabricPool connected to a couple aggr's on an A900.  Will these metrics show reads from the  FabricPool tier?

**Comment by cgrinds** [comment, marked as answer]
hi @JerryCurl you might already know this, but in case you don't, `bin/zapi` interrogates ONTAP for the metadata associated with those counters and shows a bit of context (probably not enough in this case).

Replace `$POLLER` with your poller name from `harvest.yml`

`bin/zapi --poller $POLLER show counters --object wafl -w color` or if you have [dasel](https://github.com/tomwright/dasel) installed `bin/zapi --poller $POLLER show counters --object wafl | dasel -r xml -w json`

Metadata ONTAP returns for the walf counter
```json
     {
        "base-counter": "read_io_type_base",
        "desc": "Percentage of reads served from buffer cache, ext_cache, or disk",
        "is-deprecated": "false",
        "labels": {
          "label-info": "cache,pmem,ext_cache,disk,bamboo_ssd,hya_hdd,hya_cache,hya_non_cache,cloud,fc_miss,cloud_s2c"
        },
        "name": "read_io_type",
        "privilege-level": "diag",
        "properties": "percent",
        "type": "array",
        "unit": "percent"
      },
```

**Reply by cgrinds** [reply]
We're also have a pull request (https://github.com/NetApp/harvest/pull/1352) in review that adds four FabricPool panels to the Volume dashboard.

Those panels are consuming metrics from the ONTAP counter `wafl_comp_aggr_vol_bin`.
